src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        Lotery lotery = new Lotery();
+        for (int i = 0; i < 3; i++) {
+            balls[i] = lotery.getRandomBall();
+            System.out.println(balls[i].toString());
+        }
```

Comment:
3 is a magic number, make it a constant
Also, I think we can remove the array and just generate a ball and print it right away, we can even remove the `toString()` call, since it will be called automatically if we just pass that ball to sout like that:
```java
System.out.println(lottery.getRandomBall());
```